sunpy/map/sources/rhessi.py

```diff
@@ -4,33 +4,43 @@
 __author__ = "Steven Christe"
 __email__ = "steven.d.christe@nasa.gov"
 
-import matplotlib.pyplot as plt
-
 from sunpy.map import GenericMap
+from sunpy.cm import cm
 
 import astropy.units as u
 
 __all__ = ['RHESSIMap']
 
 class RHESSIMap(GenericMap):
-    """RHESSI Image Map definition
+    """RHESSI Image Map
+
+    The RHESSI mission consists of a single spin-stabilized
+    spacecraft in a low-altitude orbit inclined 38 degrees to
+    the Earth's equator. The only instrument on board is an
+    Germaniun imaging spectrometer with the ability to obtain high
+    fidelity solar images in X rays (down to 3 keV) to gamma rays (1 MeV).
+
+    RHESSI provides an angular resolution of 2 arcseconds at
+    X-ray energies below ~40 keV, 7 arcseconds to 400 keV,
+    and 36 arcseconds for gamma-ray lines and continuum above 1 MeV.
```

Comment:
Is this information in a journal somewhere, or is the RHESSI homepage the relevant reference?